src/cmd_context/tactic_cmds.cpp

```diff
@@ -675,12 +679,14 @@ tactic * sexpr2tactic(cmd_context & ctx, sexpr * n) {
             return mk_and_then(ctx, n);
         else if (cmd_name == "or-else")
             return mk_or_else(ctx, n);
+#ifndef SINGLE_THREAD
```

Comment:
too many changes.
Let the tactic creation throw. You will not be using them in single threaded build anyhow.

Reply:
I can create a special `util/thread.h` header with `ifndef` in it.